pkg/handler/interactive.go

```diff
@@ -43,14 +44,9 @@ func NewInteractiveHandler(sess ssh.Session, user *model.User, jmsService *servi
 	return handler
 }
 
-var (
-	// 全局永久缓存 ssh 登录用户切换的语言
-	userLangGlobalStore = sync.Map{}
-)
-
 func getUserDefaultLangCode(user *model.User) string {
-	if langCode, ok := userLangGlobalStore.Load(user.ID); ok {
-		return langCode.(string)
+	if user.Language != "" {
+		return user.Language
 	}
 	return config.GetConf().LanguageCode
 }
```

Comment:
There seems to be no syntax errors or unexpected behavior in the provided code snippet. It is quite concise already.

However, it can help optimize this function:

function getUserDefaultLangCode() {
  var userLang := config.GetConf().LanguageCode

  if !utils.IsNilOrEmptyString(userLang) {
    return userLang
  }

  // Check if user has set their default language (e.g., 'zh', 'en')
  // If yes, use that instead of global store
  if utils.IsValidLanguageName(user.DefaultLocale) {
  	userLang = common.NormalizeText(user.DefaultLocale)
  }

  // Set the default locale in global map for future look-ups with `Load()` method
  if utils.SetGlobalMap("koko-user-default-lang", user.Id(), userLang)

  return userLang
}